### DISCUSSION TITLE
tools:replace error

### DISCUSSION THREAD

**Discussion by DevVigilante** [Sketchware-Pro/Sketchware-Pro#734] [Q&A] [answered]
Maybe a future was added to Sketchware Pro in previous releases of Sketchware pro which was used to remove property of wedget  (eg id property) in inject is not working.If we want to remove more than 1 attributes in xml we see an error while compiling project of duplicate attribute as follow
`/storage/emulated/0/.sketchware/mysc/606/app/src/main/res/layout/main.xml:27: error: duplicate attribute.
/storage/emulated/0/.sketchware/mysc/606/app/src/main/res/layout/main.xml: error: file failed to compile.
`

**Comment by j-px** [comment]
Can you show that View's inject property? Or even better, upload main.xml (AS FILE!) here?

**Comment by DevVigilante** [comment]
[Link](https://dl.dropbox.com/s/0fh9t5h1gm95411/2023-1-29T153644%20NewProject%20v1.0%20%28com.my.newproject%2C%201%29%202023-1-29T153644.swb?dl=0)

**Comment by j-px** [comment, marked as answer]
```xml
tools:replace="android:orientation"
tools:replace="android:padding"
```
is not valid XML. Just as the error said, that's duplicate attributes! Use it like this instead:
```
tools:replace="android:orientation, android:padding"
```
